src/locales/en/pages.yml

```diff
@@ -12,19 +12,16 @@ landingPage:
       two: "Then select which Post Office you want to take your photo ID to."
       three: "We'll send you a confirmation email with a link to download your Post Office customer letter."
     secondStep:
-      one: "Take your Post Office customer letter and photo ID to your chosen Post Office. "
+      one: "Take your Post Office customer letter and photo ID to your chosen Post Office. Or go to any "
```

Comment:
This is because that part of the sentence is a plain string but the following words are a link to the Post Office website
<img width="618" alt="Screenshot 2023-07-26 at 11 13 11" src="https://github.com/alphagov/di-ipv-cri-f2f-front/assets/117987734/d963feee-f85a-4fe6-9b7f-2a060eedd48e">